cookbook/controller/upload_file.rst

```diff
@@ -146,16 +159,26 @@ Finally, you need to update the code of the controller that handles the form::
         }
     }
 
+Now, create the ``brochures_directory`` parameter that was used in the
+controller to specify the directory in which the brochures should be stored:
+
+.. code-block:: yaml
+
+    # app/config/parameters.yml
+    parameters:
+        # ...
+        brochures_directory: '%kernel.root_dir%/../web/uploads/brochures'
```

Comment:
And actually, we're dependent on it later when we make the paths to it in the Twig template.